huaweicloud/services/dbss/resource_huaweicloud_dbss_instance.go

```diff
@@ -277,20 +278,20 @@ func buildPayOrderBodyParams(orderId string) map[string]interface{} {
 	return bodyParams
 }
 
-func buildCreateInstanceBodyParams(d *schema.ResourceData, config *config.Config) map[string]interface{} {
+func buildCreateInstanceBodyParams(d *schema.ResourceData, cfg *config.Config) map[string]interface{} {
 	bodyParams := map[string]interface{}{
-		"region":                config.GetRegion(d),
+		"region":                cfg.GetRegion(d),
 		"name":                  utils.ValueIngoreEmpty(d.Get("name")),
 		"comment":               utils.ValueIngoreEmpty(d.Get("description")),
 		"availability_zone":     utils.ValueIngoreEmpty(d.Get("availability_zone")),
 		"cloud_service_type":    "hws.service.type.dbss",
 		"flavor_ref":            utils.ValueIngoreEmpty(d.Get("flavor")),
 		"vpc_id":                utils.ValueIngoreEmpty(d.Get("vpc_id")),
-		"nics":                  buildCreateInstanceRequestBodyCreateInstanceRequestBody_nics(d),
-		"security_groups":       buildCreateInstanceRequestBodyCreateInstanceRequestBody_security_groups(d),
-		"product_infos":         buildCreateInstanceRequestBodyCreateInstanceRequestBody_product_infos(d),
+		"nics":                  buildCreateInstanceRequestBodyCreateInstanceRequestBodyNics(d),
+		"security_groups":       buildCreateInstanceRequestBodyCreateInstanceRequestBodySecurityGroups(d),
+		"product_infos":         buildCreateInstanceRequestBodyCreateInstanceRequestBodyProductInfos(d),
```

Comment:
it's better to make the function name shorter and clearer.
```suggestion
		"nics":                  buildCreateInstanceNicsRequestBody(d),
		"security_groups":       buildCreateInstanceSecurityGroupsRequestBody(d),
		"product_infos":         buildCreateInstanceProductInfosRequestBody(d),
```